grails-app/domain/org/broadinstitute/orsp/IssueExtraProperty.groovy

```diff
@@ -63,9 +63,7 @@ class IssueExtraProperty  implements LogicalDelete<IssueExtraProperty> {
     public static final String PII = "pii"
     public static final String COMPLIANCE = "compliance"
     public static final String TEXT_COMPLIANCE = "textCompliance"
-    public static final String SENSITIVE = "sensitive"
     public static final String SENSITIVE_INFORMATION_SOURCE = "sensitiveInformationSource"
-    public static final String TEXT_SENSITIVE = "textSensitive"
```

Comment:
nit: It would be nice to get rid of these in dev. Not a requirement for this PR (i.e. no liquibase job needed), but we should manually clean these up.